### PR TITLE
bump sitemap-generator gem version

### LINKED ISSUE #3: Sitemaps fail to generate
**Describe the bug**
Sitemaps fail to generate, workarea 3.5.9

**To Reproduce**
1. `rails c`
2. `Workarea::GenerateSitemaps.perform_async`
```ruby
irb(main):002:0> Workarea::GenerateSitemaps.perform_async
In '/Users/path/to/project/tmp/sitemaps/':
Traceback (most recent call last):
        1: from (irb):2
NoMethodError (undefined method `yaml_as' for SitemapGenerator::BigDecimal:Class)
Did you mean?  yaml_tag
```

**Expected behavior**
```ruby
irb(main):001:0>  Workarea::GenerateSitemaps.perform_async
In '/Users/path/to/project/tmp/sitemaps/':
+ sitemap.xml.gz                                        1593 links /       8 KB
Sitemap stats: 1,593 links / 1 sitemaps / 0m01s
=> "0e3d662d11101c529b233f0f"
```

**Workarea Setup:**
 - Workarea Version: [e.g. v3.5.9]
 - Plugins
```ruby
# Open source Workarea gems
source 'https://rubygems.org/' do
  gem 'workarea', '3.5.9'
  gem 'workarea-basic_auth', '1.1.2'
  gem 'workarea-blog', '3.5.0'
  gem 'workarea-braintree', '1.1.1'
  gem 'workarea-classic_search_autocomplete', '1.0.0'
  gem 'workarea-gdpr', '1.1.0'
  gem 'workarea-google_analytics', '2.2.1'
  gem 'workarea-google_tag_manager', '4.3.2'
  gem 'workarea-mail_chimp', '3.1.0'
  gem 'workarea-sitemaps', '2.1.7'
  gem 'workarea-slick_slider', '1.0.3'
  gem 'workarea-styled_selects', '1.1.2'
  gem 'workarea-super_hero', '2.3.0'
  gem 'workarea-variant_list'
end

# Private Workarea gems
source 'https://gems.weblinc.com' do
  gem 'workarea-avatax', '4.2.1'
  gem 'workarea-b2b', '1.1.3'
  gem 'workarea-category_overview', '1.0.1'
  gem 'workarea-ci', '3.4.14'
  gem 'workarea-commerce_cloud', '~> 1.2.2'
  gem 'workarea-expandable_content_block', '1.0.0'
  gem 'workarea-oms', '5.3.1'
  gem 'workarea-product_quickview', '2.0.4'
  gem 'workarea-quick_order', '1.0.0'
  gem 'workarea-returns', '1.2.1'
  gem 'workarea-sentry', '1.3.3'
  gem 'workarea-slider_block', '1.2.0'
end

gem 'workarea-upgrade', source: 'https://gems.weblinc.com', group: 'development'

group :qa, :staging, :production do
  gem 'newrelic_rpm'
end
```

### PR DESCRIPTION
**Pull request by alexx-avdeev**
Fixes #3 

**Describe the bug**
Sitemaps fail to generate, workarea 3.5.9

**To Reproduce**
1. `rails c`
2. `Workarea::GenerateSitemaps.perform_async`
```ruby
irb(main):002:0> Workarea::GenerateSitemaps.perform_async
In '/Users/path/to/project/tmp/sitemaps/':
Traceback (most recent call last):
        1: from (irb):2
NoMethodError (undefined method `yaml_as' for SitemapGenerator::BigDecimal:Class)
Did you mean?  yaml_tag
```

**Expected behavior**
```ruby
irb(main):001:0>  Workarea::GenerateSitemaps.perform_async
In '/Users/path/to/project/tmp/sitemaps/':
+ sitemap.xml.gz                                        1593 links /       8 KB
Sitemap stats: 1,593 links / 1 sitemaps / 0m01s
=> "0e3d662d11101c529b233f0f"
```

**Workarea Setup:**
 - Workarea Version: [e.g. v3.5.9]
 - Plugins
```ruby
# Open source Workarea gems
source 'https://rubygems.org/' do
  gem 'workarea', '3.5.9'
  gem 'workarea-basic_auth', '1.1.2'
  gem 'workarea-blog', '3.5.0'
  gem 'workarea-braintree', '1.1.1'
  gem 'workarea-classic_search_autocomplete', '1.0.0'
  gem 'workarea-gdpr', '1.1.0'
  gem 'workarea-google_analytics', '2.2.1'
  gem 'workarea-google_tag_manager', '4.3.2'
  gem 'workarea-mail_chimp', '3.1.0'
  gem 'workarea-sitemaps', '2.1.7'
  gem 'workarea-slick_slider', '1.0.3'
  gem 'workarea-styled_selects', '1.1.2'
  gem 'workarea-super_hero', '2.3.0'
  gem 'workarea-variant_list'
end

# Private Workarea gems
source 'https://gems.weblinc.com' do
  gem 'workarea-avatax', '4.2.1'
  gem 'workarea-b2b', '1.1.3'
  gem 'workarea-category_overview', '1.0.1'
  gem 'workarea-ci', '3.4.14'
  gem 'workarea-commerce_cloud', '~> 1.2.2'
  gem 'workarea-expandable_content_block', '1.0.0'
  gem 'workarea-oms', '5.3.1'
  gem 'workarea-product_quickview', '2.0.4'
  gem 'workarea-quick_order', '1.0.0'
  gem 'workarea-returns', '1.2.1'
  gem 'workarea-sentry', '1.3.3'
  gem 'workarea-slider_block', '1.2.0'
end

gem 'workarea-upgrade', source: 'https://gems.weblinc.com', group: 'development'

group :qa, :staging, :production do
  gem 'newrelic_rpm'
end
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Workarea Sitemaps 2.1.8 (2020-07-01)
```
Comment (mttdffy) [inline CHANGELOG.md]: Please remove this change, this get generated dynamically when we run the release task

**Review by mttdffy — CHANGES_REQUESTED** [review]
Can you look into why there is no ci action run for this branch? Biggest factor of bumping is seeing if the test suite continues to pass. Thanks

**Comment by bencrouse** [comment]
afaik, we haven't really figured out the flow for running the build from a fork. Maybe it's because Github Actions isn't enabled for that repo? I ran the tests locally and they passed. I'm going to merge and let the full build run before releasing.

**Review by bencrouse — APPROVED** [review]

**Comment by alexx-avdeev** [comment]
> afaik, we haven't really figured out the flow for running the build from a fork. Maybe it's because Github Actions isn't enabled for that repo? I ran the tests locally and they passed. I'm going to merge and let the full build run before releasing.

I was initially trying to push the branch to the main repo, but it seems to be prohibited, so I went the fork route...

**Comment by bencrouse** [comment]
This is released in v2.1.8.

**Pull request merged**